pkg/WORKSPACE

```diff
@@ -1,4 +1,28 @@
+# Copyright 2020 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 workspace(name = "rules_pkg")
 
 load("//:deps.bzl", "rules_pkg_dependencies")
+
 rules_pkg_dependencies()
+
+load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
```

Comment:
.bzl style is to put all the loads at the top of the file.

But the secondary question is why skylib is added here rather  than in rules_pkg_dependencies?

Reply:
Never mind. I had my federation hat on.  And that comment itself probably won't make sense.